src/azure-cli/azure/cli/command_modules/storage/sdkutil.py

```diff
@@ -75,3 +75,31 @@ def get_container_access_type(cli_ctx, name):
     if name == 'container':
         return get_sdk(cli_ctx, ResourceType.DATA_STORAGE, 'PublicAccess', mod='blob.models').Container
     raise KeyError
+
+
+def get_fs_access_type_names():
+    return 'off', 'file', 'filesystem'
+
+
+def get_fs_access_type(cli_ctx, name):
+    if name == 'off':
+        return None
+    if name == 'file':
+        return get_sdk(cli_ctx, ResourceType.DATA_STORAGE_FILEDATALAKE, 'PublicAccess', mod='_models').File
+    if name == 'filesystem':
+        return get_sdk(cli_ctx, ResourceType.DATA_STORAGE_FILEDATALAKE, 'PublicAccess', mod='_models').FileSystem
+    raise KeyError
+
+
+def _dont_fail_on_exist(ex, error_code):
+    """
+    don't throw exception if the resource doesn't exist.
+    This is called by create_* APIs with fail_on_exist=False
+    :param error:
+    :param resource:
+    :return:
+    """
+    if ex.error_code == error_code:
+        return False
```

Comment:
no need to return a false value

Reply:
False will be used as output for exists command here. Finally the output will be:
```
{
    "exists": false
}
```